resources/js/pages/admin/countries/Edit.vue

```diff
@@ -0,0 +1,206 @@
+<script setup lang="ts">
+// packages
+import { Head, useForm, usePage } from '@inertiajs/vue3';
+import { toInteger } from 'lodash';
+// shadcn ui
+import { Button } from '@/components/ui/button';
+import { Input } from '@/components/ui/input';
+import { Label } from '@/components/ui/label';
+import { Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from '@/components/ui/select';
+// generated (wayfinder)
+import CountryController from '@/actions/App/Http/Controllers/Admin/CountryController';
+
+import DeleteCountry from '@/components/countries/DeleteCountry.vue';
+import InputError from '@/components/InputError.vue';
+import { can } from '@/composables/hasPermissions';
+import AppLayout from '@/layouts/AppLayout.vue';
+import type { BreadcrumbItem, Country, Currency, SharedData } from '@/types';
+
+const page = usePage<SharedData>();
+const country = page.props.country as Country;
+const currencies = page.props.currencies as Currency[];
+let currency = currencies.find((currency) => currency.id === country.currency_id) as Currency;
+
+const breadcrumbs: BreadcrumbItem[] = [
+    {
+        title: 'Countries',
+        href: CountryController.index(),
+    },
+    {
+        title: 'Edit (' + country.name + ')',
+        href: CountryController.edit(country),
+    },
+];
+
+const form = useForm({
+    currency_id: country.currency_id,
+    iso_alpha_2: country.iso_alpha_2,
+    iso_alpha_3: country.iso_alpha_3,
+    iso_numeric: country.iso_numeric,
+    name: country.name,
+});
+
+const updateCurrencyValue = (newCurrencyId: number) => {
+    currency = currencies.find((currency) => currency.id === toInteger(newCurrencyId));
+};
+
+const submit = () => {
+    form.patch(CountryController.update(country));
+};
```

Comment:
Let's see if we can move away from this and use the `Form` component like the other create and edit pages.